src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/EditorConfigValueSerializer.cs

```diff
@@ -28,6 +28,20 @@ private static string UnescapeLineBreaks(string str)
             parseValue: str => EscapeLineBreaks(str),
             serializeValue: UnescapeLineBreaks);
 
+        private static readonly EditorConfigValueSerializer<bool?> s_nullableBoolean = new(
+            parseValue: ParseNullableBoolean,
+            serializeValue: nullableBoolean => nullableBoolean == null ? "null" : nullableBoolean.Value.ToString());
```

Comment:
>.ToString( [](http://example.com/codeflow?start=103&length=10)

I think we should be consistent with `s_bool` serializer and return `value ? "true" : "false"`